src/main/java/org/kuali/kfs/module/purap/document/PurchaseOrderDocument.java

```diff
@@ -1797,8 +1780,8 @@ public PersistableBusinessObject getNoteTarget() {
         PurchaseOrderDocument oldest = docDao.findByDocumentHeaderId(PurchaseOrderDocument.class, purchaseOrderDao.getOldestPurchaseOrderDocumentNumber(this.getPurapDocumentIdentifier()));
 
         //KFSMI-9746: added this for null safe checking.
-        if(oldest != null){
-            return oldest;
+        if (oldest != null) {
+            return oldest.getDocumentHeader();
```

Comment:
I believe we've purposely customized this to return the reference to "oldest" instead of its document header. Thus, I don't think the document header should be returned here, since that could interfere with notes on our existing POs.

Reply:
Good point Chad, it looks like this was customized as part of KFSUPGRADE-762 to avoid a conversion during that upgrade.